README.md

```diff
@@ -1,4 +1,16 @@
-# track_your_needs
+# Track your needs
+The App lets you keep track of your basic needs: sleep, social life, nutrition and sport. 
+
+By filling in your needs daily and seeing a weekly overview you notice which needs to take better care of. 
+
+If a need is fulfilled the correlating need bar is green but if you for example don't get enough sleep the bar turns red.
+
+
+# Technology Stack
+Vue JS
+Google Firebase
+SASS
```

Comment:
Add "future work" section below (would be good to have if you've time).
Add screenshots (At least the one with weekly overview that you sent me).

In the screenshots you still have nutrition and sport bars. If you keep them mention in the description or future work that they're not funcitonal at the moment (otherwise it's a bit confusing for people who didn't see your project before).